source/numerics.tex

```diff
@@ -4241,8 +4228,7 @@
 \begin{itemdescr}
 \pnum
 \effects
-Constructs a \tcode{seed_seq} object
- as if by default-constructing its member \tcode{v}.
+Default-constructs its member \tcode{v}.
```

Comment:
"Default-initializes the member \tcode{v}." would be better here, but we should instead `noexcept = default` the declaration in the class synopsis and remove the detailed specification completely:
* defaulted constructors default-initialize non-static data members with no default member initializers, so we needn't spell out that `v` is default-initialized
* `vector`'s default constructor is conditionally `noexcept(noexcept(Allocator()))`, which is `true` for `std::allocator` and therefore for `v`'s type `vector<result_type>`, so this constructor *should* be `noexcept` rather than "_Throws_: Nothing." per LWG policy. (I assume no one has noticed that we could strengthen `noexcept` here since the conditional `noexcept` was added to `vector`'s default constructor.)

Reply:
Please open a separate editorial issue for that. Let's not accumulate too much burden on this one.